### ISSUE TITLE
IPython notebook crash

### ISSUE THREAD

**Issue by tvandera** [ExaScience/smurff#150]
The https://github.com/ExaScience/smurff/blob/master/docs/notebooks/different_noise_models.ipynb crashes in this test:

```
RMSE = 0.91
INFO:root:PythonSession {
  Data: {
    Type: ScarceMatrixData [with NAs]
    Component-wise mean: 6.35508
    Component-wise variance: 1.89449
    Noise: Probit Noise with threshold 6
    Size: 47424 [15073 x 346] (0.91%)
      Warning: 77 empty rows
      Warning: 1 empty cols
  }
  Model: {
    Num-latents: 32
    Dimensions: 15073,346
  }
  Priors: {
    0: NormalPrior
    1: NormalPrior
  }
  Result: {
    Test data: 11856 [15073 x 346] (0.23%)
    Binary classification threshold: 6.00
      54.92% positives in test data
  }
  Config: {
      Iterations: 100 burnin + 100 samples
      Save model: never
  }
}
```

**Comment by tvandera** [comment]
Seems to be the next one, with Probit + Macau. Goes out of memory in init phase

**Comment by tvandera** [comment]
Fixed in dca46080412a3af496d4b2caeee558b7b0abb3d2, merged in #151
